<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FOE_Main</class>
 <widget class="QMainWindow" name="FOE_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">FOE ClanManager [v0.4.1]</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="accessibleDescription">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addUserButton">
        <property name="text">
         <string>Add user</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteUserButton">
        <property name="text">
         <string>Delete user</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIndstillinger">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionForbindelse"/>
   </widget>
   <addaction name="menuIndstillinger"/>
  </widget>
  <action name="actionForbindelse">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
